include/pisa/scorer/pl2.hpp

```diff
@@ -16,13 +16,13 @@ template <typename Wand>
 struct pl2: public index_scorer<Wand> {
     using index_scorer<Wand>::index_scorer;
 
-    static constexpr float c = 1;
+    pl2(const Wand& wdata, const float c) : index_scorer<Wand>(wdata), m_c(c) {}
```

Comment:
Same idea as for BM25 only for `c`.

Reply:
Added some verbose descriptions, maybe we can clean them up.